### PR TITLE
Added `requirements.txt` install to Payu test runs

### PR DESCRIPTION
**Pull request by CodeGat**
In this PR:
* checks.yml: Added installation of a `requirements.txt` from the `*-configs` repository before Payu.

### CODE REVIEW

File: .github/workflows/checks.yml [hunk 1]
```diff
@@ -73,6 +73,8 @@ jobs:
           cd ${{ env.BASE_EXPERIMENT_LOCATION }}
           git checkout ${{ inputs.config-tag }}
 
+          pip install -r ${{ env.BASE_EXPERIMENT_LOCATION }}/test/requirements.txt
```
Comment (aidanheerdegen) [inline .github/workflows/checks.yml:76]: What happens if this doesn't exist?

**Review by aidanheerdegen — DISMISSED** [review]
LGTM, just one question

File: .github/workflows/checks.yml (hunk 1 above)
Comment (CodeGat) [reply .github/workflows/checks.yml]: Updated with error checking! :D

**Review by aidanheerdegen — APPROVED** [review]
LGTM

**Pull request merged**